From: Jiri Slaby <jslaby@suse.cz>
Date: Wed, 7 Sep 2016 18:20:09 +0200
Subject: Revert "Input: i8042 - set up shared ps2_cmd_mutex for AUX ports"
Patch-mainline: never, kabi

This reverts commit 9781b971f456189f3664d6f5bd2d5643893c0408, upstream
commit 47af45d684b5f3ae000ad448db02ce4f13f73273. It depends on commit
11dd037e42590ee224658ddddfb715e5ce1d328a, upstream
commit 4097461897df91041382ff6fcd2bfa7ee6b2448c which breaks kABI.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/input/serio/i8042.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/input/serio/i8042.c b/drivers/input/serio/i8042.c
index 405252a884dd..b4d34086e73f 100644
--- a/drivers/input/serio/i8042.c
+++ b/drivers/input/serio/i8042.c
@@ -1305,7 +1305,6 @@ static int __init i8042_create_aux_port(int idx)
 	serio->write		= i8042_aux_write;
 	serio->start		= i8042_start;
 	serio->stop		= i8042_stop;
-	serio->ps2_cmd_mutex	= &i8042_mutex;
 	serio->port_data	= port;
 	serio->dev.parent	= &i8042_platform_device->dev;
 	if (idx < 0) {
-- 
2.10.0

